Makefile

```diff
@@ -54,11 +54,31 @@ clean:
 # must be included after the default target
 -include $(BUILD_SYSTEM_DIR)/makefiles/targets.mk
 
+## Possible values: prod; debug
+TARGET ?= prod
 
 ## Git version
 GIT_VERSION ?= $(shell git describe --abbrev=6 --always --tags)
 NIM_PARAMS := $(NIM_PARAMS) -d:git_version=\"$(GIT_VERSION)\"
 
+## Heaptracker options
+HEAPTRACKER ?= 0
+HEAPTRACKER_INJECT ?= 0
+ifeq ($(HEAPTRACKER), 1)
+# Needed to make nimbus-build-system use the Nim's 'heaptrack_support' branch
+NIM_COMMIT := NIM_COMMIT=heaptrack_support
```

Comment:
Would this result in `NIM_COMMIT` having a value `NIM_COMMIT=heaptrack_support` - i.e. trying to checkout branch `NIM_COMMIT=heaptrack_support`, rather than `heaptrack_support`?

Reply:
Ok, I've applied your suggestions and I think it now looks much better. Thanks @vpavlin !